tools/generator/test/BuildSettingConditionalTests.swift

```diff
@@ -29,7 +29,24 @@ final class BuildSettingConditionalTests: XCTestCase {
         "any": .any,
     ]
 
-    func test_comparable() {
+    func test_comparable() throws {
+        // Arrange
+
+        let conditionalA = try XCTUnwrap(Self.conditionals["A"])
```

Comment:
We control the fixtures, in the same file even, so I would rather not use `XCTUnwrap`.

```suggestion
        let conditionalA = Self.conditionals["A"]!
```

Reply:
No problem. I'd argue that as the fixtures change, seeing test failures instead of test crashes is more helpful to debug and iterate, but that's a personal workflow choice.